### PR TITLE
Add mittwald extension

### PR DESCRIPTION
**Pull request by danieldeichfuss**
## Description

[mittwald](https://www.mittwald.de/) is a hosting provider in Germany. With this extension you can navigate in mittwald projects, organizations, apps, domains and the API documentation.

## Screencast

https://github.com/user-attachments/assets/8c666448-26bb-4b18-9a8d-dcf4b020c56e

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/mittwald/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# mittwald Changelog
+
+## [Initial Version] - 2024-08-21
```
Comment (pernielsentikaer) [inline extensions/mittwald/CHANGELOG.md:3]:
```suggestion
## [Initial Version] - {PR_MERGE_DATE}
```

File: extensions/mittwald/package.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "author": "danieldeichfuss",
+  "categories": [
+    "Developer Tools",
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-mittwald-projects",
+      "title": "Search Projects",
```
Comment (pernielsentikaer) [inline extensions/mittwald/package.json:16]:
```suggestion
      "title": "Search Projects",
      "subtitle": "mittwald",
```

File: extensions/mittwald/package.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      "description": "Search mittwald projects",
+      "mode": "view"
+    },
+    {
+      "name": "search-mittwald-domains",
+      "title": "Search Domains",
```
Comment (pernielsentikaer) [inline extensions/mittwald/package.json:22]:
```suggestion
      "title": "Search Domains",
      "subtitle": "mittwald",
```

File: extensions/mittwald/package.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      "description": "Search mittwald domains",
+      "mode": "view"
+    },
+    {
+      "name": "search-mittwald-organizations",
+      "title": "Search Organizations",
```
Comment (pernielsentikaer) [inline extensions/mittwald/package.json:28]:
```suggestion
      "title": "Search Organizations",
      "subtitle": "mittwald",
```

File: extensions/mittwald/package.json [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+      "description": "Search mittwald organizations",
+      "mode": "view"
+    },
+    {
+      "name": "search-mittwald-apps",
+      "title": "Search Apps",
```
Comment (pernielsentikaer) [inline extensions/mittwald/package.json:34]:
```suggestion
      "title": "Search Apps",
      "subtitle": "mittwald",
```

File: extensions/mittwald/package.json [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+      "description": "Search mittwald apps",
+      "mode": "view"
+    },
+    {
+      "name": "search-mittwald-api-endpoints",
+      "title": "Search API Endpoints",
```
Comment (pernielsentikaer) [inline extensions/mittwald/package.json:40]:
```suggestion
      "title": "Search API Endpoints",
      "subtitle": "mittwald",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by danieldeichfuss** [comment]
Hi @pernielsentikaer thanks a lot for your review. I made all the requested changes ✅

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Not sure why my last comment was not included, but you can also reference all images from `metadata` in your `README` so you can remove duplicates in media 🙂

**Comment by pernielsentikaer** [comment]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**